lms/djangoapps/experiments/utils.py

```diff
@@ -67,6 +67,25 @@
                                  flag_undefined_default=False)
 # TODO END: clean up as part of REVEM-199 (End)
 
+# TODO: Clean up as part of REV-1205 (START)
+# .. toggle_name: experiments.add_upsell_tracking
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Make sure upsell tracking JS works as expected.
+# .. toggle_category: experiments
+# .. toggle_use_cases: monitored_rollout
+# .. toggle_creation_date: 2020-7-7
+# .. toggle_expiration_date: None
+# .. toggle_warnings: None
+# .. toggle_tickets: REV-1205
+# .. toggle_status: supported
+UPSELL_TRACKING_FLAG = WaffleFlag(
+    waffle_namespace=experiments_namespace,
```

Comment:
Do you need the variable assignment here? Can you pass just `WaffleFlag(experiments_namespace, u'add_upsell_tracking', False)`?

Reply:
I copy/pasted it from the PROGRAM_INFO_FLAG definition. It looks like the WaffleFlag supports both positional params and named params, and I personally prefer the latter since it's clearer what each value is being used for.